GovUkDesignSystemComponents/TextInput.cshtml

```diff
@@ -20,7 +20,7 @@
             await Html.RenderPartialAsync("/GovUkDesignSystemComponents/ErrorMessage.cshtml", Model.ErrorMessage);
         }
     }
-    <input class="govuk-input @(Model.Classes) @(Model.ErrorMessage != null ? "govuk-input--error" : "")"
+    <input class=" @(Model.Classes ?? "govuk-input") @(Model.ErrorMessage != null ? "govuk-input--error" : "")"

```

Comment:
I don't think you need this?

Reply:
Yes you are correct. If we just need a regular input control then we set the class property to "govuk-input".